generatebundlefile/hack/generate_bundle.sh

```diff
@@ -32,7 +32,7 @@ if [ ! -x "${ORAS_BIN}" ]; then
 fi
 
 make build
-chmod +x ${BASE_DIRECTORY}/bin/generatebundlefile 
+chmod +x ${BASE_DIRECTORY}/generatebundlefile/bin
```

Comment:
execute on the directory? What perms does it have that you want to add execute? Do you have read already on it?